back/engines/commercial/impact_tracking/app/models/impact_tracking/salt.rb

```diff
@@ -4,7 +4,7 @@
 #
 # Table name: impact_tracking_salts
 #
-#  id         :bigint           not null, primary key
+#  id         :uuid             not null, primary key
```

Comment:
Why is this change here?

Reply:
I believe somebody just forgot to update the annotation as the migration has `uuid` https://github.com/CitizenLabDotCo/citizenlab/blob/9b159c63d13cc2a8864fcb88c4d002274038c547/back/db/migrate/20220927114325_create_salts.impact_tracking.rb#L12

Reply:
OK, I thought these comments got updated automatically, but it seems that is not always the case.